src/Core/Installer.php

```diff
@@ -399,15 +397,25 @@ public function checkFunctions()
 			$status = false;
 			$help = DI::l10n()->t('Error: PDO or MySQLi PHP module required but not installed.');
 			$returnVal = false;
-		} else {
-			if (!function_exists('mysqli_connect') && class_exists('pdo') && !in_array('mysql', \PDO::getAvailableDrivers())) {
-				$status = false;
-				$help = DI::l10n()->t('Error: The MySQL driver for PDO is not installed.');
-				$returnVal = false;
-			}
+		} elseif (!function_exists('mysqli_connect') && class_exists('pdo') && !in_array('mysql', \PDO::getAvailableDrivers())) {
+			$status = false;
+			$help = DI::l10n()->t('Error: The MySQL driver for PDO is not installed.');
+			$returnVal = false;
 		}
 		$this->addCheck(DI::l10n()->t('PDO or MySQLi PHP module'), $status, true, $help);
 
+		// Acticate when IntlChar is installed in the check pipeline.
```

Comment:
Wouldn't this benefit admins though, despite the check pipeline failing?
```suggestion
		// Uncomment when IntlChar is installed in the check pipeline.
```

Reply:
Of course I would like to activate it - but then the pipeline would fail all the time.